bytecode/concrete.py

```diff
@@ -283,22 +284,39 @@ def to_code(self):
         code_str, linenos = self._assemble_code()
         lnotab = self._assemble_lnotab(self.first_lineno, linenos)
         nlocals = len(self.varnames)
-        return types.CodeType(self.argcount,
-                              self.kwonlyargcount,
-                              nlocals,
-                              # FIXME: compute stack size
-                              self._stacksize,
-                              self.flags,
-                              code_str,
-                              tuple(self.consts),
-                              tuple(self.names),
-                              tuple(self.varnames),
-                              self.filename,
-                              self.name,
-                              self.first_lineno,
-                              lnotab,
-                              tuple(self.freevars),
-                              tuple(self.cellvars))
+        if sys.version_info >= (3,):
+            return types.CodeType(self.argcount,
```

Comment:
I would suggest to use an "args" list to not have to duplicate ~15 lines of code, and then use CodeType(*args).